dotnet3.5/S3/ListObjectsExample/ListObjects/ListObjects.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+
+// snippet-start:[S3.dotnet35.ListObjects]
+
+using Amazon;
+using Amazon.S3;
+using Amazon.S3.Model;
+using System;
+using System.Threading.Tasks;
+
+namespace ListObjects
+{
+    // The following example lists objects in an Amazon Simple Storage
+    // Service (Amazon S3) bucket. It was created using AWS SDK for .NET 3.5
+    // and .NET 5.0.
+    public class ListObjects
+    {
+        // Specify your AWS Region (an example Region is shown).
+        private static readonly RegionEndpoint BUCKET_REGION = RegionEndpoint.USWest2;
+        private static IAmazonS3 _s3Client;
+
+        private const string BUCKET_NAME = "doc-example-bucket";
+
+        static void Main()
```

Comment:
I think it's more modern/conventional now to use `static async Task Main`, and then we can `await` on line 30 instead of `.Wait()`. 

I believe this is more asynchronous, see the **Is “await task;” the same thing as “task.Wait()”?** section from https://devblogs.microsoft.com/pfxteam/asyncawait-faq/

Reply:
I've made the change to make Main asynchronous. I will also apply this to my other examples.